change/react-native-windows-extended-2019-09-04-17-03-52-testdll-def-CreateStringResponseBody.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "none",
+  "comment": "Update CreateStringResponseBody symbol in test DLL.",
+  "packageName": "react-native-windows-extended",
```

Comment:
This only has x86 and x64. Do we build arm as well?

Reply:
Not at this point.